dask_expr/tests/test_collection.py

```diff
@@ -123,6 +123,8 @@ def test_conditionals(func, pdf, df):
         lambda df: df.apply(lambda row, x, y=10: row * x + y, x=2),
         lambda df: df[df.x > 5],
         lambda df: df.assign(a=df.x + df.y, b=df.x - df.y),
+        lambda df: df.isna(),
+        lambda df: df.x.isna(),
```

Comment:
We may want `pdf` to have at least one null for features like this. Or perhaps we have a `pdf_nulls` fixture?